src/main/java/org/elasticsearch/action/bulk/BulkRequest.java

```diff
@@ -417,7 +417,7 @@ public final BulkRequest timeout(TimeValue timeout) {
      * A timeout to wait if the index operation can't be performed immediately. Defaults to <tt>1m</tt>.
      */
     public final BulkRequest timeout(String timeout) {
-        return timeout(TimeValue.parseTimeValue(timeout, null));
+        return timeout(TimeValue.parseTimeValue(timeout, null, "BulkRequest.timeout"));
```

Comment:
just a suggestion, would it make sense to use `BulkRequest.class.getSimpleName() + ".timeout"` I am afraid we missing this stuff in refactorings